### ISSUE TITLE
HuBERT example: 'ConvolutionalPositionalEmbedding' object has no attribute 'kernel_size'

### ISSUE THREAD

**Issue by pyf98** [pytorch/audio#2845]
### 🐛 Describe the bug

I tried to pre-train the HuBERT model using the provided example, but encountered this error:

```
Traceback (most recent call last):
  File "/usr0/home/yifanpen/torchaudio_examples/hubert/train.py", line 223, in <module>
    cli_main()
  File "/usr0/home/yifanpen/torchaudio_examples/hubert/train.py", line 219, in cli_main
    run_train(args)
  File "/usr0/home/yifanpen/torchaudio_examples/hubert/train.py", line 64, in run_train
    model = HuBERTPreTrainModule(
  File "/usr0/home/yifanpen/torchaudio_examples/hubert/lightning.py", line 142, in __init__
    self.model = torchaudio.models.hubert_pretrain_base(
  File "/usr0/home/yifanpen/miniconda3/envs/torchaudio_ssl/lib/python3.10/site-packages/torchaudio/models/wav2vec2/model.py", line 1039, in hubert_pretrain_base
    return hubert_pretrain_model(
  File "/usr0/home/yifanpen/miniconda3/envs/torchaudio_ssl/lib/python3.10/site-packages/torchaudio/models/wav2vec2/model.py", line 995, in hubert_pretrain_model
    model.apply(_init_hubert_pretrain_model)
  File "/usr0/home/yifanpen/miniconda3/envs/torchaudio_ssl/lib/python3.10/site-packages/torch/nn/modules/module.py", line 726, in apply
    module.apply(fn)
  File "/usr0/home/yifanpen/miniconda3/envs/torchaudio_ssl/lib/python3.10/site-packages/torch/nn/modules/module.py", line 726, in apply
    module.apply(fn)
  File "/usr0/home/yifanpen/miniconda3/envs/torchaudio_ssl/lib/python3.10/site-packages/torch/nn/modules/module.py", line 726, in apply
    module.apply(fn)
  [Previous line repeated 1 more time]
  File "/usr0/home/yifanpen/miniconda3/envs/torchaudio_ssl/lib/python3.10/site-packages/torch/nn/modules/module.py", line 727, in apply
    fn(self)
  File "/usr0/home/yifanpen/miniconda3/envs/torchaudio_ssl/lib/python3.10/site-packages/torchaudio/models/wav2vec2/model.py", line 690, in _init_hubert_pretrain_model
    std = math.sqrt(4.0 / (module.embed_dim * module.kernel_size))
  File "/usr0/home/yifanpen/miniconda3/envs/torchaudio_ssl/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1265, in __getattr__
    raise AttributeError("'{}' object has no attribute '{}'".format(
AttributeError: 'ConvolutionalPositionalEmbedding' object has no attribute 'kernel_size'
```

In [`_init_hubert_pretrain_model`](https://github.com/pytorch/audio/blob/main/torchaudio/models/wav2vec2/model.py#L690), `module.kernel_size` is used, but this attribute does not exist in the [`ConvolutionalPositionalEmbedding`](https://github.com/pytorch/audio/blob/main/torchaudio/models/wav2vec2/components.py#L185) module.

### Versions

Collecting environment information...
PyTorch version: 1.13.0
Is debug build: False
CUDA used to build PyTorch: 11.7
ROCM used to build PyTorch: N/A

OS: Ubuntu 20.04.5 LTS (x86_64)
GCC version: (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0
Clang version: Could not collect
CMake version: version 3.16.3
Libc version: glibc-2.31

Python version: 3.10.6 (main, Oct 24 2022, 16:07:47) [GCC 11.2.0] (64-bit runtime)
Python platform: Linux-5.15.0-52-generic-x86_64-with-glibc2.31
Is CUDA available: True
CUDA runtime version: 11.7.99
CUDA_MODULE_LOADING set to: LAZY
GPU models and configuration: 
GPU 0: NVIDIA RTX A5000
GPU 1: NVIDIA RTX A5000
GPU 2: NVIDIA RTX A5000
GPU 3: NVIDIA RTX A5000
GPU 4: NVIDIA RTX A5000
GPU 5: NVIDIA RTX A5000
GPU 6: NVIDIA RTX A5000
GPU 7: NVIDIA RTX A5000

Nvidia driver version: 520.61.05
cuDNN version: Could not collect
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True

Versions of relevant libraries:
[pip3] numpy==1.23.3
[pip3] pytorch-lightning==1.8.1
[pip3] torch==1.13.0
[pip3] torchaudio==0.13.0
[pip3] torchmetrics==0.10.2
[conda] blas                      1.0                         mkl  
[conda] mkl                       2021.4.0           h06a4308_640  
[conda] mkl-service               2.4.0           py310h7f8727e_0  
[conda] mkl_fft                   1.3.1           py310hd6ae3a3_0  
[conda] mkl_random                1.2.2           py310h00e6091_0  
[conda] numpy                     1.23.3          py310hd5efca6_1  
[conda] numpy-base                1.23.3          py310h8e6c178_1  
[conda] pytorch                   1.13.0          py3.10_cuda11.7_cudnn8.5.0_0    pytorch
[conda] pytorch-cuda              11.7                 h67b0de4_0    pytorch
[conda] pytorch-lightning         1.8.1                    pypi_0    pypi
[conda] pytorch-mutex             1.0                        cuda    pytorch
[conda] torchaudio                0.13.0              py310_cu117    pytorch
[conda] torchmetrics              0.10.2                   pypi_0    pypi

**Comment by nateanl** [comment]
Hi @pyf98, thanks for reporting the issue. It was caused by #2716. I will make a fix for it.

**Comment by pyf98** [comment]
Thanks for the quick fix!

**Comment by nateanl** [comment]
Fixed in #2846, let me know if you meet any other issue.
